packages/xsnap/src/xsnap.js

```diff
@@ -87,7 +157,7 @@ export async function xsnap(options) {
     netstringMaxChunkSize = undefined,
     parserBufferSize = undefined,
     snapshotStream,
```

Comment:
Given our findings in https://gist.github.com/mhofman/7fbfd71048148b97089993bb4463b465 I am slightly conflicted about keeping this binding in the surrounding scope of `makeSnapshotStream`, but then the only major engine that seems to not support split scopes (JSC) would also need `options` to be nulled too, or skipped altogether (deconstructed in the params)